libs/SalesforceSDK/src/com/salesforce/androidsdk/security/BiometricAuthenticationManager.kt

```diff
@@ -127,7 +124,7 @@ internal class BiometricAuthenticationManager: AppLockManager(
         internal const val BIO_AUTH_NATIVE_BUTTON = "bio_auth_native_button"
         internal const val SHOW_BIOMETRIC = "show_biometric"
 
-        fun isEnabled(userAccount: UserAccount): Boolean {
+        fun isBiometricAuthenticationEnabled(userAccount: UserAccount): Boolean {
```

Comment:
Since this is a (static) companion method (and we often import the companion object) I found making it more descriptive made reading the code a lot clear.